spec/lib/log_parser_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "log_parser"
```

Comment:
```suggestion
require "log_parser"

```

Espaço para ficar mais legível